ChangeLog

```diff
@@ -20,6 +20,7 @@ master 8.16
 - webpsave has @passes parameter to set number of passes to achieve
   desired target_size [john-parton]
 - revise sRGB -> B_W coefficients [daniellovera]
+- fix pipe read limit
```

Comment:
Is it worth mentioning that the previous default limit of 1GB has been removed (and must now be set explicitly if required)?

Reply:
It looks like the default is still 1GiB, unless I'm overlooking something:
https://github.com/libvips/libvips/blob/9de5f24dab4ea2064666b351960a9719f27a4ac5/libvips/iofuncs/source.c#L79-L84

Reply:
D'oh, yes it is, thank you Kleis.